pact/Pact/Core/Errors.hs

```diff
@@ -1225,32 +1225,32 @@ ensureBound msg = BoundedText (T.take (fromIntegral (natVal (Proxy @k))) msg)
 dbOpErrorToBoundedText' :: DbOpError -> Text
 dbOpErrorToBoundedText' = \case
     WriteError ->
-      "Error found while writing value"
+      "Database error: Writing " rk "in" tn " failed because of an error."
     RowReadDecodeFailure rk ->
-      thsep ["Failed to deserialize but found value at key:", abbrevText 10 rk]
+      thsep ["Deserialization failed for the value at key:", abbrevText 10 rk]
     RowFoundError tn rk ->
       thsep
-        ["Value already found while in Insert mode in table"
+        ["Insert failed because the value already exists in the table:" "
         , renderTableName tn, "at key"
         , tdquotes $ abbrevRowKey rk]
     NoRowFound tn rk ->
       thsep
-        ["No row found during update in table"
+        ["Update failed because no row was found in table:"
         , renderTableName tn
         , "at key"
         , tdquotes $ abbrevRowKey rk ]
     NoSuchTable tn ->
-      thsep ["Table", renderTableName tn, "not found"]
+      thsep ["Insert/Update failed because table ", renderTableName tn, "was not found."]
     TableAlreadyExists tn ->
-      thsep ["Table", renderTableName tn, "already exists"]
+      thsep [""Insert/Update failed because table", renderTableName tn, "already exists.""]
```

Comment:
```suggestion
      thsep [""Creating of user table", renderTableName tn, "failed because it already exists."]
```

Reply:
I can't commit the suggestion but with a minor tweak this looks good to me (I'm not seeing the error messages in context, but if table creation fails in this case, the message looks right to me).